### ISSUE TITLE
compile on windows

### ISSUE THREAD

**Issue by SabraHashemi** [AD-Vega/qarv#27]
I noticed that you did not mention windows but Aravis can be compile on windows,
this project is not compatible with windows?

**Comment by exzombie** [comment]
Interesting, I did not know that Aravis now works on Windows. That was not the case a couple of years ago when I last checked. There is in principle no reason why qarv would not work on Windows. There are two points that would need to be addressed:
- `getmtu_linux.h` is Linux-specific. However, the `getMTU()` function could be removed now because Aravis now has the necessary functionality itself. To just test building on windows, this function can be replaces with a dummy.
- An installer would need to be created.

So in short, nothing is preventing you from trying to build and use qarv on Windows. Please report if you are successful. Then we can take a look at supporting Windows "properly".
